grails-app/services/org/pih/warehouse/invoice/PrepaymentInvoiceService.groovy

```diff
@@ -13,6 +13,40 @@ class PrepaymentInvoiceService {
 
     InvoiceService invoiceService
 
+    Invoice generatePrepaymentInvoice(Order order) {
```

Comment:
where are these methods being called? I don't see any other controllers

Reply:
In the `generatePrepaymentInvoice` action in the `InvoiceController`